<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="30" failed="2" ignored="0" total="35" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="SOI-66" duration-ms="122793" started-at="2020-01-03T15:19:47Z" finished-at="2020-01-03T15:21:50Z">
    <groups>
    </groups>
    <test name="Salesforce Direct Sales" duration-ms="122055" started-at="2020-01-03T15:19:47Z" finished-at="2020-01-03T15:21:49Z">
      <class name="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement">
        <test-method status="PASS" signature="step01()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step01" duration-ms="30790" started-at="2020-01-03T15:20:55Z" finished-at="2020-01-03T15:21:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step01 -->
        <test-method status="PASS" signature="step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step02" duration-ms="12421" started-at="2020-01-03T15:21:26Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step01" finished-at="2020-01-03T15:21:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step02 -->
        <test-method status="FAIL" signature="step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step03" duration-ms="3467" started-at="2020-01-03T15:21:39Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step02" finished-at="2020-01-03T15:21:42Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Test Failed on Step 3]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Test Failed on Step 3
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step03(AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.java:173)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.Exception: Validation Failed on Step 3
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step03(AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.java:167)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step03 -->
        <test-method status="SKIP" signature="step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step04" duration-ms="0" started-at="2020-01-03T15:21:42Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step03" finished-at="2020-01-03T15:21:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step04 -->
        <test-method status="SKIP" signature="step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step05" duration-ms="0" started-at="2020-01-03T15:21:42Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step04" finished-at="2020-01-03T15:21:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step05 -->
        <test-method status="SKIP" signature="step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step06" duration-ms="0" started-at="2020-01-03T15:21:42Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step05" finished-at="2020-01-03T15:21:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step06 -->
        <test-method status="SKIP" signature="step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="step07" duration-ms="0" started-at="2020-01-03T15:21:42Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step06" finished-at="2020-01-03T15:21:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_EditAgreement.step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step07 -->
      </class> <!-- delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement -->
      <class name="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="beforeTest" is-config="true" duration-ms="9143" started-at="2020-01-03T15:19:47Z" finished-at="2020-01-03T15:19:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="beforeTest" is-config="true" duration-ms="5624" started-at="2020-01-03T15:19:56Z" finished-at="2020-01-03T15:20:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="step01()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="step01" duration-ms="31462" started-at="2020-01-03T15:20:02Z" finished-at="2020-01-03T15:20:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step01 -->
        <test-method status="FAIL" signature="step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="step02" duration-ms="21899" started-at="2020-01-03T15:20:34Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step01" finished-at="2020-01-03T15:20:55Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Test Failed on Step 2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Test Failed on Step 2
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step02(AgreementCreation_From_Oportunity_MobileVoice.java:152)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.Exception: Validation Failed on Step 2
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step02(AgreementCreation_From_Oportunity_MobileVoice.java:146)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step02 -->
        <test-method status="SKIP" signature="step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="step03" duration-ms="0" started-at="2020-01-03T15:20:55Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step02" finished-at="2020-01-03T15:20:55Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step03 -->
        <test-method status="SKIP" signature="step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="step04" duration-ms="0" started-at="2020-01-03T15:20:55Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step03" finished-at="2020-01-03T15:20:55Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step04 -->
        <test-method status="SKIP" signature="step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="step05" duration-ms="0" started-at="2020-01-03T15:20:55Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step04" finished-at="2020-01-03T15:20:55Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step05 -->
        <test-method status="SKIP" signature="step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="step06" duration-ms="0" started-at="2020-01-03T15:20:55Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice.step05" finished-at="2020-01-03T15:20:55Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step06 -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice@91161c7]" name="afterTest" is-config="true" duration-ms="2023" started-at="2020-01-03T15:21:42Z" finished-at="2020-01-03T15:21:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_EditAgreement@604ed9f0]" name="afterTest" is-config="true" duration-ms="5148" started-at="2020-01-03T15:21:44Z" finished-at="2020-01-03T15:21:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice -->
    </test> <!-- Salesforce Direct Sales -->
    <test name="Salesforce Partners Community" duration-ms="690" started-at="2020-01-03T15:21:49Z" finished-at="2020-01-03T15:21:50Z">
      <class name="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC">
        <test-method status="SKIP" signature="step01()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step01" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:157)
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.beforeTest(AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step01 -->
        <test-method status="SKIP" signature="step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step02" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC.step01" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step02 -->
        <test-method status="SKIP" signature="step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step03" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC.step02" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step03 -->
        <test-method status="SKIP" signature="step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step04" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC.step03" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step04 -->
        <test-method status="SKIP" signature="step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step05" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC.step04" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step05 -->
        <test-method status="SKIP" signature="step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step06" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC.step05" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step06 -->
        <test-method status="SKIP" signature="step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="step07" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC.step06" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_SFPC.step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step07 -->
      </class> <!-- delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC -->
      <class name="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC">
        <test-method status="SKIP" signature="step01()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step01" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:157)
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.beforeTest(AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step01 -->
        <test-method status="SKIP" signature="step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step02" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step01" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step02 -->
        <test-method status="SKIP" signature="step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step03" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step02" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step03 -->
        <test-method status="SKIP" signature="step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step04" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step03" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step04 -->
        <test-method status="SKIP" signature="step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step05" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step04" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step05 -->
        <test-method status="SKIP" signature="step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step06" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step05" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step06 -->
        <test-method status="SKIP" signature="step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step07" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step06" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step07()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step07 -->
        <test-method status="SKIP" signature="step08()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step08" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step07" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step08()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step08()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step08 -->
        <test-method status="SKIP" signature="step09()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="step09" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step08" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step09()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC.step09()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step09 -->
      </class> <!-- delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC -->
      <class name="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC">
        <test-method status="FAIL" signature="beforeTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="beforeTest" is-config="true" duration-ms="561" started-at="2020-01-03T15:21:49Z" finished-at="2020-01-03T15:21:50Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:157)
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.beforeTest(AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="beforeTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="beforeTest" is-config="true" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="beforeTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="beforeTest" is-config="true" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="step01()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="step01" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: cannot create default profile directory
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INNOWAVE-383', ip: '192.168.100.249', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
Ordinal0 [0x00A6A903+1550595]
Ordinal0 [0x009EA701+1025793]
Ordinal0 [0x0096C6E5+509669]
Ordinal0 [0x008FF009+61449]
Ordinal0 [0x008FC01C+49180]
Ordinal0 [0x0091ACE7+175335]
Ordinal0 [0x0091A8ED+174317]
Ordinal0 [0x00918CDB+167131]
Ordinal0 [0x0090144A+70730]
Ordinal0 [0x009024D0+74960]
Ordinal0 [0x00902469+74857]
Ordinal0 [0x00A042C7+1131207]
GetHandleVerifier [0x00B070FD+523789]
GetHandleVerifier [0x00B06E90+523168]
GetHandleVerifier [0x00B0E1E7+552695]
GetHandleVerifier [0x00B078FA+525834]
Ordinal0 [0x009FB7FC+1095676]
Ordinal0 [0x00A0633B+1139515]
Ordinal0 [0x00A064A3+1139875]
Ordinal0 [0x00A05425+1135653]
BaseThreadInitThunk [0x76CD6359+25]
RtlGetAppContainerNamedObjectPath [0x770E7B74+228]
RtlGetAppContainerNamedObjectPath [0x770E7B44+180]

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:157)
at delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.beforeTest(AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step01 -->
        <test-method status="SKIP" signature="step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="step02" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step01" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step02()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step02 -->
        <test-method status="SKIP" signature="step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="step03" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step02" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step03()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step03 -->
        <test-method status="SKIP" signature="step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="step04" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step03" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step04()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step04 -->
        <test-method status="SKIP" signature="step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="step05" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step04" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step05()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step05 -->
        <test-method status="SKIP" signature="step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="step06" duration-ms="0" started-at="2020-01-03T15:21:50Z" depends-on-methods="delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step05" finished-at="2020-01-03T15:21:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC.step06()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step06 -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC@73ad2d6]" name="afterTest" is-config="true" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_SFPC@7085bdee]" name="afterTest" is-config="true" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:delivery01.soi_66.AgreementCreation_From_Opportunity_MobileVoice_EditAgreement_SFPC@1ce92674]" name="afterTest" is-config="true" duration-ms="0" started-at="2020-01-03T15:21:50Z" finished-at="2020-01-03T15:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- delivery01.soi_66.AgreementCreation_From_Oportunity_MobileVoice_Negative_SFPC -->
    </test> <!-- Salesforce Partners Community -->
  </suite> <!-- SOI-66 -->
</testng-results>
